### ISSUE TITLE
Decided dataset for pre-training

### ISSUE THREAD

**Issue by crangelsmith** [alan-turing-institute/heartfm-data#1]
Options listed in [CINEMA](https://github.com/mathpluscode/CineMA?tab=readme-ov-file#-using-pre-trained-models).

ACDC	[README.md](https://github.com/mathpluscode/CineMA/blob/main/cinema/data/acdc/README.md)
M&Ms	[README.md](https://github.com/mathpluscode/CineMA/blob/main/cinema/data/mnms/README.md)
M&Ms2	[README.md](https://github.com/mathpluscode/CineMA/blob/main/cinema/data/mnms2/README.md)
Kaggle	[README.md](https://github.com/mathpluscode/CineMA/blob/main/cinema/data/kaggle/README.md)
Rescan	[README.md](https://github.com/mathpluscode/CineMA/blob/main/cinema/data/rescan/README.md)
Landmark	[README.md](https://github.com/mathpluscode/CineMA/blob/main/cinema/data/landmark/README.md)
EMIDEC	[README.md](https://github.com/mathpluscode/CineMA/blob/main/cinema/data/emidec/README.md)
Myops2020	[README.md](https://github.com/mathpluscode/CineMA/blob/main/cinema/data/myops2020/README.md)

Are there any other cine-MRI datasets available?

**Comment by myyong** [comment]
There are some datasets and challenges as well, on the [Cardiac Atlas Project](https://www.cardiacatlas.org/).

**Comment by LevanBokeria** [comment]
MICCAI runs [CMRxRecon challenges](https://cmrxrecon.github.io/2025/Home.html) where they share raw MRI files (called k space files) from which the anatomical normal MRI images are reconstructed. 

<img width="1701" height="643" alt="Image" src="https://github.com/user-attachments/assets/5bc7ea03-3eec-4633-bfba-0578ec1027ec" />

Seems like they have a good amount of subjects over the years. Each year contains train, val, and test sets. The validation sets do not have the full k-space images, but only subsampled ones. 
So counting the training and test images from all 3 challenges, theres total of 500+270+250=**1020 subjects.** 
The data does contain cine MRI images of the short axis of the heart. 

Current obstacles:
- Downloading data from 2023/2024 requires getting permission from admins. Might be easy as emailing. 
- The 2025 challenge is ongoing and the data has an embargo for non-challenge use till Dec 2025. Might get around this if we ask. Wenja Bai is [one of the organisers](https://cmrxrecon.github.io/2025/Organizer.html)
- Without looking at the data, its not clear whether the SA cine cardiac images are single slice or multi slice. Also not clear if they are acquired only at one timepoint of the cardiac cycle, or throughout it. So we might have as little as just 1020 images (I doubth it) or as many as 100k-400k.
- The MRI images are in k space. To get the real anatomical images, they need to be inverse Fourier transformed. This should be doable, but an extra step that might uncover unexpected issues.
- Will require a MATLAB license to reconstruct k-space images to real-space images. We can get a 1 month free trial though.

**Comment by LevanBokeria** [comment]
### EMIDEC
I've downloaded and preprocessed the EMIDEC dataset. The preprocessing generates metadata about the images.
From that metadata, I estimate that with about 150 subjects (train and test), we would have **~1000 SAX images.**

### Landmark
For the Landmark dataset, we need to contact Rhodri Davies ([rhodri.davies@ucl.ac.uk](mailto:rhodri.davies@ucl.ac.uk)).

From the CineMA paper: 

> 4.5 Data for long-axis function estimation 
> The landmark dataset, which contains 1,444 studies with LAX 2C and 4C images at both ED and ES phases, was used to estimate long-axis function.

So this would get us 1444 * 2 * 2 = **5776 LAX images.** 

### MyOps2020

I've preprocessed the data as well. The metadata says for both train and test folders, we'd have total of **174 images**.

**Comment by LevanBokeria** [comment]
### Rescan: 

We'd need to contact Rhodri Davies again ([rhodri.davies@ucl.ac.uk](mailto:rhodri.davies@ucl.ac.uk)).
There is a website too for requesting access to data, asking for details on intended use: https://thevolumesresource.com/

Details about the number of subjects and scans are available [here](https://thevolumesresource.com/resource-information/)
See the table at the end for scan acquisition characteristics.

In short:
- 50 test-retest scans (so 100 scans, I think?) available by just registering on their website.
- Another 60 test-retest scans (so 120 scans) available if we collaborate with them.

We could assume ~9 SAX slices acquired each time point, and ~25 frames acquired over the cardiac cycle (based on the temporal resolution they state in the table).
This would give us:
- 100 scans * 9 slices * 25 frames = **22,500 images.**
- 120 scans * 9 slices * 25 frames = **27,000 images.**

If "50 test-retest scans" means just 50 scans (so 25 pairs), then these numbers should be halved. 

I am not sure if we'd need a DSA or something when they share the scans.

**Comment by LevanBokeria** [comment]
### Kaggle:

n subjects: 500 training, 200 validation, and 440 test.
Each subject seems to have multiple types of scans: 10 SAX slices, 2CH LAX slice, and a 4CH LAX slice, so total of 12 slices. Each of those slices is acquired 30 times across a hearbear. So thats a lot of images:

Training: 500 * 12 * 30 = **180k**
Validation: 200 * 12 * 30 = **72k**
Test: 440 * 12 * 30 = **158,400**

The dataset description does say there is some variation in number of slices, but the final number shouldn't be too far from those. 

The data seems freely available. I'm in the process of downloading.
